mobility_data/forms.py

```diff
@@ -6,10 +6,14 @@
 
 
 class CustomDataSourceForm(forms.ModelForm):
-    # Instead of the type name, display the more informative importer name to the user.
+    # Instead of the type name, display the more informative importer or the optional display name to the user.
     type_names = [
-        (key, value["importer_name"]) for key, value in DATA_SOURCE_IMPORTERS.items()
+        (key, value["display_name"])
+        if "display_name" in value
```

Comment:
if does what exactly?

Reply:
if display_name in value,  the type_name element will be the key, the display_name defined in DATA_SOURCE_IMPORTERS in constants.py, otherwise the value will be the importer_name defined in the DATA_SOURCE_IMPORTERS.

Reply:
Oh yes, lazy reviewer did not open the whole file.